src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+@import "../util/variables.scss";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $max-width-calendar;
+  gap: $gap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    height: $square-size;
+    width: $square-size;
+    border: $border-width solid #000;
+    transition: all 0.5s ease;
+    cursor: pointer;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+  }
+
+  @each $day-of-weak, $day-num in $start-day {
+    &--start-day-#{$day-of-weak} {
```

Comment:
you should to add this margin not for calendar but for first element of calendar 
![image](https://github.com/mate-academy/layout_calendar/assets/96265844/88d78d62-92ab-4ff2-a921-5984d48a548e)